### DISCUSSION TITLE
Fire HD Tablets 2021 - supported?

### DISCUSSION THREAD

**Discussion by Staini** [AppGround-io/bluetooth-keyboard-and-mouse-support#192] [Q&A] [answered]
The next black friday is soon. So I would like to ask if the new Fire HD tablets are fully supported?

Fire HD 10 (3GB) and Fire HD 10 Plus (4GB)
11. Generation 2021
Fire OS 7 (Android 9, API Level 28)

**Comment by AppGround-io** [comment, marked as answer]
Fire OS 7 only supports the compatibility connection mode, so there can be connection problems. 
If there is a Fire OS 8 in the near future it will hopefully will support the normal connection mode and the new devices most likely will get the update.

**Reply by Staini** [reply]
Thanks for your detailed answer.
Then I will probably buy one soon and I keep my fingers crossed for Fire OS 8.
